docs/src/main/paradox/bootstrap/demos/kubernetes-api.md

```diff
@@ -54,7 +55,7 @@ $ sbt shell
 
 Once the image is published, you can deploy it onto the kubernetes cluster by calling:
 
-@@snip [kube-create.sh](../../../../../bootstrap-joining-demo/kubernetes-api/kube-create.sh) 
+@@snip [kube-create.sh]($management$/bootstrap-demo/kubernetes-api/kube-create.sh) 
 
 This will create and start running a number of Pods hosting the application. The application nodes will proceed with 
 forming the cluster using the DNS bootstrap method. In order to observe the logs during the cluster formation you can 
```

Comment:
Just noticed that this page talks about the DNS bootstrap method, but it's supposed to be the Kubernetes API demo.

Reply:
Raised  https://github.com/akka/akka-management/issues/292 to track making that example purely k8s API